main.pm

```diff
@@ -498,7 +498,7 @@ sub load_consoletests() {
         loadtest "console/zypper_ref.pm";
         loadtest "console/yast2_lan.pm";
         loadtest "console/curl_https.pm";
-        if (!get_var("OFW")) {
+        if (!get_var("OFW") && check_var("ARCH", "x86_64")) {
```

Comment:
Yes, we don't want this to be run on aarch64 either. The check makes only sense on x86_64